### PR TITLE
Telemetry guide: made editorial, formatting, and spelling revisions.

### PR DESCRIPTION
**Pull request by DougTW**
Signed-off-by: DougTW <doug.martin@intel.com>

### CODE REVIEW

File: source/clear-linux/guides/telemetrics/telem-guide.rst [hunk 1]
```diff
@@ -378,7 +491,8 @@ Confirm that the telemetrics header file is located on the system at :file:`usr/
     #include <telemetry.h>
 
 
-   Use the following code to create the variables we need to hold the data for the record we will be creating:
+   Use the following code to create the variables needed to hold the data for
+   the record to be created:
 
    ::
```
Comment (mvincerx) [inline source/clear-linux/guides/telemetrics/telem-guide.rst]: Wherever, you see `::` in this document, please replace it with our standard 

"""
.. code-block:: console
   
   my_code_here 
   ....      
 """

This document follows the above convention until Ln 449; thereafter, it changes to use `::`.

File: source/clear-linux/guides/telemetrics/telem-guide.rst [hunk 2]
```diff
@@ -393,27 +507,37 @@ Confirm that the telemetrics header file is located on the system at :file:`usr/
 
    Severity:
     | Type: uint32_t
-    | Value:  Severity field value. Accepted values are in the range 1-4, with 1 being the lowest severity, and 4 being the highest severity. Values provided outside of this range are clamped to 1 or 4. [low, med, high, crit]
+    | Value:  Severity field value. Accepted values are in the range 1-4, with
```
Comment (mvincerx) [inline source/clear-linux/guides/telemetrics/telem-guide.rst]: Please remove the single `|`, or pipe, character. It's non-standard, and it causes build-errors. Reformat these sections. For these headers, like Severity, please use **Severity** and check each one of the header's left-alignment to assure it aligns with the previous paragraph above, or instruction--and not the code-block. I can walk you through the fussier parts of this if you like.

Comment (mvincerx) [reply source/clear-linux/guides/telemetrics/telem-guide.rst]: I experimented with correcting the `::` and removing the `|` characters to achieve a clean build. Just takes a little work.

File: source/clear-linux/guides/telemetrics/telem-guide.rst (hunk 1 above)
Comment (DougTW) [reply source/clear-linux/guides/telemetrics/telem-guide.rst]: I'll fix it and commit ASAP.

File: source/clear-linux/guides/telemetrics/telem-guide.rst (hunk 2 above)
Comment (DougTW) [reply source/clear-linux/guides/telemetrics/telem-guide.rst]: Removed pipes and left-aligned subsequent text to header left margin.

File: source/clear-linux/guides/telemetrics/telem-guide.rst (hunk 1 above)
Comment (DougTW) [reply source/clear-linux/guides/telemetrics/telem-guide.rst]: Replaced old code notation using double colon (::) with code-block console notation.

**Comment by mvincerx** [comment]
@DougTW. Please hold per our chat. I'll push to this branch.

**Comment by mvincerx** [comment]
LGTM.

**Pull request merged**